lib/ansible/plugins/cliconf/aireos.py

```diff
@@ -19,6 +19,16 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+DOCUMENTATION = """
+---
+cliconf: aireos
+short_description: Use aireos cliconf to run command on Cisco WLC platform
```

Comment:
These don't match? Should we pick one?

Reply:
looks like the modules both use Cisco WLC so lets go with that unless you thing AireOS is more well-known.

Reply:
:+1: